### ISSUE TITLE
Overwriting the variable `sys.path`

### ISSUE THREAD

**Issue by skupr-anaconda** [Nuitka/Nuitka#2221]
Is it correct that `sys.path` variable https://github.com/nuitka/nuitka/commit/1765ffce2a9ab859853210337390de242cd80712#diff-bc83f19b087e9fb792a4d4b92209d6991f350e09511d958383d478f6f1c7bef9R46 is overwritten tree lines below https://github.com/nuitka/nuitka/commit/1765ffce2a9ab859853210337390de242cd80712#diff-bc83f19b087e9fb792a4d4b92209d6991f350e09511d958383d478f6f1c7bef9R49 in this [commit](https://github.com/nuitka/nuitka/commit/1765ffce2a9ab859853210337390de242cd80712#diff-bc83f19b087e9fb792a4d4b92209d6991f350e09511d958383d478f6f1c7bef9R49) related to CVE-2022-2054?

**Comment by kayhayen** [comment]
Yes, that is intentional. Without that the `ast` module could not be imported.

**Comment by skupr-anaconda** [comment]
Thanks
